packages/communication-react/CHANGELOG.beta.md

```diff
@@ -4,91 +4,57 @@ This log was last generated on Wed, 04 Oct 2023 17:17:11 GMT and should not be m
-- Introduces cap to the number of participants in the grid view when a audio only call ([PR #3574](https://github.com/azure/communication-ui-library/pull/3574) by 94866715+dmceachernmsft@users.noreply.github.com)
-- Update available layouts for composites ([PR #3637](https://github.com/azure/communication-ui-library/pull/3637) by 94866715+dmceachernmsft@users.noreply.github.com)
-- Add raise hand feature to stable ([PR #3548](https://github.com/azure/communication-ui-library/pull/3548) by 79475487+mgamis-msft@users.noreply.github.com)
-- Add e2e testing for mobile gallery option selection ([PR #3611](https://github.com/azure/communication-ui-library/pull/3611) by 94866715+dmceachernmsft@users.noreply.github.com)
-- The Web UI Chat and CallWithChat composite now supports the Image Gallery feature. This enables users to click on an individual inline image in a message bubble and display the selected image in full screen or download the image. We are also introducing a new standalone ImageGallery component to display an image in large scale with the ability to download it. ([PR #3646](https://github.com/azure/communication-ui-library/pull/3646) by 73612854+palatter@users.noreply.github.com)
-- Add optimal video count feature to stable ([PR #3539](https://github.com/azure/communication-ui-library/pull/3539) by 79475487+mgamis-msft@users.noreply.github.com)
-- Introduces new Large gallery mode to the video gallery ([PR #3591](https://github.com/azure/communication-ui-library/pull/3591) by 94866715+dmceachernmsft@users.noreply.github.com)
-- GA Captions ([PR #3547](https://github.com/azure/communication-ui-library/pull/3547) by carolinecao@microsoft.com)
-- Add capabilities feature to stable ([PR #3540](https://github.com/azure/communication-ui-library/pull/3540) by 79475487+mgamis-msft@users.noreply.github.com)
-- UI Library now supports new Gallery Layouts in the CallComposite and CallWithChatComposite on web and mobile. Users can select the new layouts from the control bar. ([PR #3645](https://github.com/azure/communication-ui-library/pull/3645) by 94866715+dmceachernmsft@users.noreply.github.com)
-- Introduce gallery controls to mobile for default and dynamic views ([PR #3603](https://github.com/azure/communication-ui-library/pull/3603) by 94866715+dmceachernmsft@users.noreply.github.com)
-- Introduce storybook docs for gallery layouts ([PR #3616](https://github.com/azure/communication-ui-library/pull/3616) by 94866715+dmceachernmsft@users.noreply.github.com)
+- Add gallery layout feature 
+[PR #3637](https://github.com/azure/communication-ui-library/pull/3637)
+[PR #3603](https://github.com/azure/communication-ui-library/pull/3603) 
+[PR #3611](https://github.com/azure/communication-ui-library/pull/3611)
+[PR #3574](https://github.com/azure/communication-ui-library/pull/3574)
+- Add image gallery feature
+[PR #3507](https://github.com/Azure/communication-ui-library/pull/3507)
```

Comment:
Do we need to list out all the PRs related to image gallery like the `Add gallery layout feature` did?

Reply:
So the issue here is that most of the image gallery PRs happened before the 1.8.0 release. So they actually weren't included in this file. After talking with Josh, he suggested just to include this one as you could use the feature flag to identify the code related to the feature.

Reply:
I have brought this up as a topic to discuss for the release retro.